code/parse/sexp.cpp

```diff
@@ -14110,6 +14120,8 @@ void sexp_add_background_bitmap(int n, bool is_sun)
 	eval_angles(&sle.ang, n, is_nan, is_nan_forever);
 	if (is_nan || is_nan_forever)
 		return;
+	if (!uses_correct_angles)
+		stars_correct_background_angles(&sle.ang);
```

Comment:
Similar to parsing above, I wonder if it might be worth writing to the debug log whether we're making this correction and whether it's for a sun or for a background.

OTOH that's equivalent to logging whether the mission uses the deprecated SEXPs for adding suns/backgrounds, the new ones, or (eep) a mix.

Reply:
Same as above - I don't think a debug statement is necessary.